Source: written by Pierre Lejeune for Exherbo
Upstream: no
Reason: Install pc file into /usr/${LIBDIR}

--- a/src/controllib/Makefile.am	2005-09-27 14:54:20.000000000 +0200
+++ b/src/controllib/Makefile.am	2012-03-23 21:43:31.702518995 +0100
@@ -19,5 +19,5 @@
 libacr38ucontrol_la_CFLAGS = `pkg-config libpcsclite --cflags`
 
 #pcdir= $(PKG_PCSC_CONFIG_PATH)
-pcdir= @prefix@/lib/pkgconfig
+pcdir= @libdir@/pkgconfig
 pc_DATA = libacr38ucontrol.pc
